targets/targets.json

```diff
@@ -3215,6 +3215,8 @@
             "STM32H7",
             "STM32H743xI"
         ],
+        "mbed_rom_start": "0x08000000",
+        "mbed_rom_size" : "0x200000",
```

Comment:
I am sorry, but in order to remove warnings,
we should also add (for both H743ZI and ZI2):
"mbed_ram_start": "0x24000000",
"mbed_ram_size" : "0x80000",

Reply:
Which warnings? I don't recall seeing any?

Reply:
Done, anyways.